src/starklings_runner.rs

```diff
@@ -105,20 +83,40 @@ pub fn run_cairo_program(args: &Args) -> anyhow::Result<String> {
     )
     .with_context(|| "Failed setting up runner.")?;
     let result = runner
-        .run_function(
+        .run_function_with_starknet_context(
             runner.find_function("::main")?,
             &[],
             args.available_gas,
             StarknetState::default(),
         )
         .with_context(|| "Failed to run the function.")?;
     match result.value {
-        cairo_lang_runner::RunResultValue::Success(values) => ret_string
-            .push_str(format!("Run completed successfully, returning {values:?}").as_str()),
+        cairo_lang_runner::RunResultValue::Success(values) => {
```

Comment:
Have you verified that this works correctly? I think using the `ret_string` variable was required here to get the output of the run 

I tried an my previous solution for `enum2.cairo` is not compiling anymore - but the error message doesn't appear

Reply:
Ohh, I tested with some arbitrary cairo and it seemed to compile/show error. Lemme check enum example.

Reply:
I reverted to old signatures but it was still happening,

This seemed to fix the issue.

![image](https://github.com/shramee/starklings-cairo1/assets/11048263/2e781940-581e-49cc-b7e0-8b1e95a69a8a)